src/Ast/Type/CallableTypeTemplateNode.php

```diff
@@ -0,0 +1,35 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\PhpDocParser\Ast\Type;
+
+use PHPStan\PhpDocParser\Ast\Node;
+use PHPStan\PhpDocParser\Ast\NodeAttributes;
+
+class CallableTypeTemplateNode implements Node
+{
+
+	use NodeAttributes;
+
+	/** @var IdentifierTypeNode */
```

Comment:
`TemplateTagValueNode` names it `name` of type `string`, @ondrejmirtes do you want apply any of these?